### PR TITLE
add gperf, libbsd, libmd, libpthread-stubs

### PR DESCRIPTION
**Pull request by rawlingsj**
Signed-off-by: James Rawlings <jrawlings@chainguard.dev>

### CODE REVIEW

**Comment by rawlingsj** [comment]
this is the start of the JDK dependencies.

File: gperf.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: GPL-3.0-or-later
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/bootstrap/stage3
```
Comment (kaniini) [inline gperf.yaml:16]: Stage3 should be dropped here.

File: gperf.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/bootstrap/stage3/wolfi-signing.rsa.pub
```
Comment (kaniini) [inline gperf.yaml:19]: Likewise.  Also, for future reference, the signing key is redundant here (both are the same key).

**Review by kaniini — CHANGES_REQUESTED** [review]
These look mostly fine, except for the stage3 references, which need to be removed.

File: gperf.yaml (hunk 1 above)
Comment (rawlingsj) [reply gperf.yaml:16]: great - will sort that now

**Review by kaniini — APPROVED** [review]

**Pull request merged**